lib/internal/bootstrap/node.js

```diff
@@ -271,6 +271,24 @@ function startup() {
     'process.assert() is deprecated. Please use the `assert` module instead.',
     'DEP0100');
 
+  // TODO(joyeecheung): this property has not been well-maintained, should we
+  // deprecate it in favor of a better API?
+  const { isDebugBuild, hasOpenSSL } = internalBinding('config');
+  Object.defineProperty(process, 'features', {
+    enumerable: true,
+    writable: false,
+    configurable: false,
+    value: {
+      debug: isDebugBuild,
+      uv: true,
+      ipv6: true,  // TODO(bnoordhuis) ping libuv
+      tls_alpn: hasOpenSSL,
```

Comment:
This should also be  available as `process.versions.openssl` even though that's not the purpose of that property - I wonder if we should just use that instead?